zap/src/main/java/org/parosproxy/paros/Constant.java

```diff
@@ -769,6 +780,28 @@ private void initializeFilesAndDirectories(ControlOverrides overrides) {
         messages = new I18N(locale);
     }
 
+    boolean acquireHomeLock() {
+        try {
+            Path lockFile = Paths.get(zapHome, ".homelock");
+            if (Files.notExists(lockFile)) {
+                Files.createFile(lockFile);
+            }
+            homeLockFileChannel = FileChannel.open(lockFile, StandardOpenOption.WRITE);
+            homeLock = homeLockFileChannel.tryLock();
+            if (homeLock == null) {
+                System.err.println(
+                        "The home is already in use. Ensure no other ZAP instances are running with the same home: "
```

Comment:
Should this and the one just below be i18n?

Reply:
We do not i18n the start up errors.